Src/Web/Web.Shared.Net/PostSamplingTelemetryProcessor.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Microsoft.ApplicationInsights.Web
```

Comment:
can we use a namespace like ....extensibility.implementation so that we can change/remove it if needed, without changing major version?
@SergeyKanzhelev knows best about this.